relay-event-normalization/src/normalize/processor.rs

```diff
@@ -915,7 +916,7 @@ fn normalize_performance_score(
                     measurements.insert(
                         "score.total".to_owned(),
                         Measurement {
-                            value: score_total.into(),
+                            value: (score_total / weight_total).into(),
```

Comment:
Not actionable for this PR, but to not forget about if/how we can remove the `score.total` measurement.